src/home/homework.c

```diff
@@ -17,11 +17,13 @@
 #include "homework-mount.h"
 #include "homework-pkcs11.h"
 #include "homework.h"
+#include "libcrypt-util.h"
 #include "main-func.h"
 #include "memory-util.h"
 #include "missing_magic.h"
 #include "modhex.h"
 #include "mount-util.h"
+#include "libcrypt-util.h"
```

Comment:
Not fixed yet.

Reply:
Thank chunk doesn't appear in the latest version: https://github.com/systemd/systemd/pull/16981/commits/0e98d17e77a024a16fc15589b2b21bb6196d4567#diff-bcb9e659df93349604b67a5db839f0b6 ?

Reply:
The commit includes it correctly, but 2b7a18056726543f493a2e0a3bcb6d997a7ed1b3 needlessly includes it again...